index.bs

```diff
@@ -5508,8 +5513,8 @@ the string “OrNull”.
     is written as <code class="idl">boolean?</code>:
 
     <pre highlight="webidl">
-        interface MyConstants {
-          const boolean? ARE_WE_THERE_YET = false;
+        interface NetworkFetcher {
+          void get(optional boolean? areWeThereYet = false);
```

Comment:
Why is the constant gone from this example?

Reply:
This example is used to illustrate how nullable types work, not how constants work. Plus, https://github.com/heycam/webidl/issues/446#issuecomment-331388615.

Reply:
Oops. Sorry. I didn't know we had const scattered across examples in the spec. :-/